frontend/sw360-portlet/src/main/webapp/html/projects/detail.jsp

```diff
@@ -44,7 +44,11 @@
     <div id="header"></div>
     <p class="pageHeader"><span class="pageHeaderBigSpan">Project: <sw360:ProjectName project="${project}"/></span>
         <span class="pull-right">
-        <input type="button" id="edit" value="Edit" class="addButton">
+        <input type="button" id="edit" value="Edit" class="addButton"
+            <c:if test = "${(project.clearingState eq 'CLOSED') && (isUserAdmin != 'Yes')}">
+                <c:out value="disabled='disabled'"/>
```

Comment:
Minor comment: escaping via `<c:out` is unnecessary:
```suggestion
                disabled="disabled"
```

Reply:
The `<c:out` tag is usually used to access fields in java objects, e.g. something like `<c:out value="customer.address.street"/>`. (see https://www.tutorialspoint.com/jsp/jstl_core_out_tag). This means this code interprets `disabled='disabled'` as an expression.

This should be removed.

Reply:
Done